jni-gen/src/generators/module.rs

```diff
@@ -43,6 +43,8 @@ pub fn generate_module(class_names: Vec<&ClassName>) -> String {
     vec![
         "//! Automatically generated code\n".to_string(),
         "#![allow(non_snake_case)]\n".to_string(),
+        "#[path = \"builtins/utils.rs\"]".to_string(),
```

Comment:
```suggestion
```
If you add a `mod.rs` file to the builtins folder, then you can import it without having to specify a path.